### ISSUE TITLE
VTS doesn't support connecting to milvus with TLS

### ISSUE THREAD

**Issue by gifi-siby** [zilliztech/vts#32]
This is how we are connecting to Milvus and we are using self signed certificates for authentication
```
uri = "https://ibmlhadmin:password@localhost:19530"
user = "user"
password ="password"
client = MilvusClient(
                      uri=uri,
                      user=user,
                      password=password,
                      secure=True,
                      db_name="default",
                      server_pem_path='/root/tls/cert.crt',
                      server_name='localhost',)
```

So, I am unable to modify VTS configurations to adhere the certificates as well.

**Comment by nianliuu** [comment]
Yes, It's not support for now, we will support it in the next release

**Comment by stephen37** [comment]
We should have a new release by next week 🤞

**Comment by sphinxrave** [comment]
I'm afraid the TLS implementation is incomplete? it seems to lack support for `client_pem_path` described in two-way TLS connection https://milvus.io/docs/tls.md#Two-way-TLS-connection

Would also appreciate guidance on where to put the tls files, right next to the migration.conf in the CWD or in /milvus/tls or somewhere else.

**Comment by nianliuu** [comment]
@sphinxrave I added this new config, it's supported now
